TASK1/score.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+if (isset($_POST['submit'])) {
+  // If submitted then split the text area string based on new lines.
+  $subjects = explode("\n", $_POST['score']);
+  // Display the table for scores.
+  echo '<table class="marks" cellspacing="10">
```

Comment:
Ask around how to use HTML in PHP.
We should not be echoing the entire HTML like this. This is wrong.